### PR TITLE
[OC-575] Add support for displaying custom banner-alert icon

### PR DESCRIPTION
**Pull request by edmgt**
+ [X] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack/blob/master/CONTRIBUTING.md)

_If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/master/CODE_REVIEW_GUIDELINES.md)_

---
Adds a new optional property `customBannerIcon` to `bpk-component-banner-alert` component which allows to pass a custom icon as a component.

Example how it would look like with CurrencyIcon:
image

### CODE REVIEW

File: UNRELEASED.md [hunk 1]
```diff
@@ -24,6 +24,9 @@ See [`CHANGELOG.md`](CHANGELOG.md) for real-world examples of good changelog ent
 
 - bpk-component-infinity-gauntlet:
   - New `timeStone` prop for controlling time. See &lt;link to docs site&gt;.
+
```
Comment (georgegillams) [inline UNRELEASED.md]: Could you put this entry just below line 3?

Comment (edmgt) [reply UNRELEASED.md]: updated 😛

File: packages/bpk-component-banner-alert/README.md [hunk 2]
```diff
@@ -107,6 +107,7 @@ const BannerAlertExpandableState = withBannerAlertState(BpkBannerAlertExpandable
 | animateOnLeave     | bool                   | false    | false         |
 | bannerClassName    | string                 | false    | null          |
 | show               | bool                   | false    | true          |
+| customBannerIcon   | func                   | false    | null          |
```
[the remark this answers is not in the document]
Comment (edmgt) [reply packages/bpk-component-banner-alert/README.md]: updated

File: UNRELEASED.md [hunk 3]
```diff
@@ -2,6 +2,11 @@
 
 > Place your changes below this line.
 
+**Added:**
+
+- bpk-component-banner-alert:
+  - New `customBannerIcon` prop for showing a custom banner icon.
```
[the remark this answers is not in the document]
Comment (edmgt) [reply UNRELEASED.md]: updated

**Pull request merged**